packages/framework/aqueduct/src/container-services/containerServices.ts

```diff
@@ -13,14 +13,21 @@ import {
 } from "@fluidframework/runtime-utils";
 
 // TODO: should this just be "s"?
+/**
+ * @deprecated See {@link @fluidframework/synthesize#DependencyContainer}
```

Comment:
Would be nice to add clarification to these notes saying that `DependencyContainer` is meant to replace usage of these APIs